packages/docs/verite/patterns/travel-rule.md

```diff
@@ -2,53 +2,104 @@
+
+The trust establishment step (which some might think of as an "anti-spam"/"anti-DoS" step to avoid spam or data-collecting crawlers) is not included in the simplified prototype in our repo. There are various prototyping efforts happening in the DIF and elsewhere that may prove adequate for this step, so we are deferring research on it in case open-source specifications get hardened in the meantime for this kind of authentication and public trust registry building.
+
+1. If in "**Suspicious Mode**", Bob's financial institution would attempt to authenticate Alice's FI, the author of M1 before responding. If in "**Fast Mode**, it could **skip to step 3**.
+2. Bob's financial institution would authenticate Alice's FI by the identifier(s) provided in M1. The simplest way of achieving this would be a `did:web` identifier expressing a DID document corresponding to the private key used to sign M1, particularly if said `did:web` identifier could be used to query public registries of regulated financial institutions maintained by trusted intermediaries, competent authorities, or your friendly local trade association and open source standards organization (i.e. Verite!).  Crucially, however, these registries should be multiple and open, to maximize spread and bootstrap an open ecosystem that is inclusive of multiple closed ones and proprietary authorities.
+3. Bob's financial institution verifies the originator's proof-of-control.
+4. Bob's financial institution constructs message "M2" which is a VC proving control over the beneficiary address.
+5. Bob's FI transmits M2 privately to Alice's FI using the callback URL specified in M1.
+6. Alice's FI receives and verifies the VC in M2, leaving both Alice's FI and Bob's FI (a) certain that each controls the designated addresses and (b) with a private secure network connection established through the specified callback URL.
+
 
-Before the transaction is broadcast, the sending and receiving FIs use the chain-agnostic mempool to publish and observe messages establishing and requesting proof of address control. These messages contain no customer data and no specific transaction data. Since the transaction amount is not included, observers are unable to determine information about the planned transaction other than its intended origin and destination (this is less information than is available in a typical mem pool, in which the actual monetary transaction is broadcast). This allows counterparty endpoints to find and connect to one another privately before exchanging counterparty information, and the proof of address credentials prevent other parties from spoofing the originator or beneficiary.
+### Part 3: Exchange of Counterparty Credentials
 
-### Part 2: Exchange of Counterparty Credentials
+If the transaction does not meet the threshold required for the Travel Rule, then Part 3 is not necessary and does not apply, as Alice's FI can simply proceed to broadcasting the blockchain transaction. 
```

Comment:
The threshold is determined by the Alice's FI considering regulatory requirements for Alice's FI and Bob's FI

Reply:
good point!